pnpm-lock.yaml

```diff
@@ -5674,7 +5674,7 @@ packages:
       '@babel/core': 7.20.12
       '@babel/preset-env': 7.20.2_@babel+core@7.20.12
       '@babel/types': 7.20.7
-      '@storybook/csf': 0.0.2-next.8
+      '@storybook/csf': 0.0.2-next.10
```

Comment:
I ran `pnpm i` and storybook internally uses `"@storybook/csf": "next"` in their `@storybook/components` package.json. I'm guessing that was why. Since it's a patch, I think it's okay

Reply:
> Since it's a patch, I think it's okay

Famous last words 😅

In gonfalon we got bit by a patch update to a dependency way back when, and made the call at the time to _always_ use fixed version numbers. (The tradeoff is that you need to stay on top of "good" patches manually. We felt that was preferable to the alternative.)

Reply:
Hahah you got me there - I agree for dependencies that are relied on by the shipped npm code. This is just for the local storybook.

Reply:
Hypothetically, could this break our live storybook? Or will a step in our build process catch it if it breaks?

Reply:
Yea, the chromatic build in this branch would be affected if the storybook build process failed for any reason